### PR TITLE
AMWSUP-17 Ansible Hub documentation link broken

### PR DESCRIPTION
**Pull request by RanabirChakraborty**
Issue: https://issues.redhat.com/browse/AMWSUP-17

EAP PR: https://github.com/ansible-middleware/wildfly/pull/161
JWS PR: https://github.com/ansible-middleware/jws/pull/258
keycloak PR: https://github.com/ansible-middleware/keycloak/pull/127
Infinispan PR: https://github.com/ansible-middleware/infinispan/pull/35
AMQ PR: https://github.com/ansible-middleware/amq/pull/107
jbcs PR: https://github.com/ansible-middleware/jbcs/pull/4

### CODE REVIEW

File: playbooks/amq_broker.yml [hunk 1]
```diff
@@ -35,8 +35,14 @@
...
     downstream_placeholder_delete:
       - build_status
+      - roles_paths
     # anything inside <!--start {{ <downstream_placeholder>_content }} --> and <!--end {{ <downstream_placeholder>_content }} --> will be replaced with content
     downstream_placeholder_content:
+      - placeholder: roles_paths
+        content: |
+          ### Roles:
+
+          * [`activemq`](https://console.redhat.com/ansible/automation-hub/repo/published/redhat/amq_broker/content/role/amq_broker/): Perform installation and configuration
```
Comment (guidograzioli) [inline playbooks/amq_broker.yml]: the replacement happens after subsitutions, so this should be `* [amq_broker]` not `* [activemq]`

File: playbooks/data_grid.yml [hunk 2]
```diff
@@ -43,8 +43,15 @@
...
     downstream_placeholder_delete:
       - build_status
+      - roles_paths
     # anything inside <!--start {{ <downstream_placeholder>_content }} --> and <!--end {{ <downstream_placeholder>_content }} --> will be replaced with content
     downstream_placeholder_content:
+      - placeholder: roles_paths
+        content: |
+          ### Included roles
+
+          * [`infinispan`](https://console.redhat.com/ansible/automation-hub/repo/published/redhat/data_grid/content/role/data_grid/): performs an installation of Infinispan or DataGrid nodes or cluster, with configuration of static caches.
```
Comment (guidograzioli) [inline playbooks/data_grid.yml]: same as above, data_grid and data_grid_cache

File: playbooks/sso.yml [hunk 3]
```diff
@@ -58,8 +58,15 @@
     # anything inside <!--start {{ <downstream_placeholder>_delete }} --> and <!--end {{ <downstream_placeholder>_delete }} --> will be removed
     downstream_placeholder_delete:
       - build_status
+      - roles_paths
     # anything inside <!--start {{ <downstream_placeholder>_content }} --> and <!--end {{ <downstream_placeholder>_content }} --> will be replaced with content
     downstream_placeholder_content:
+      - placeholder: roles_paths
+        content: |
+          ### Included roles
+          * [`keycloak`](https://console.redhat.com/ansible/automation-hub/repo/published/redhat/sso/content/role/sso/): role for installing the service.
```
Comment (guidograzioli) [inline playbooks/sso.yml]: `sso`, `sso_realm` and `sso_quarkus`

**Review by guidograzioli — CHANGES_REQUESTED** [review]

**Comment by RanabirChakraborty** [comment]
Thanks! @guidograzioli Did the required changes.

**Review by guidograzioli — CHANGES_REQUESTED** [review]
Thanks for fixing, but I am sorry there is another needed change that I didn't see yesterday. `roles_paths` goes in the placeholder_delete OR in the  placeholder_content, not both

**Comment by RanabirChakraborty** [comment]
@guidograzioli I actually tested it and it worked fine for EAP/wildfly, it actually deleting the content inside [roles_paths](https://github.com/ansible-middleware/wildfly/commit/b19cc4e7bf3648122c488968745104cdcf2c692c#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L30-L39) and replacing it with [this](https://github.com/ansible-middleware/janus/pull/38/files#diff-a0685f63ed3654ca257627c0ba578c989dbdf3f02020fc9b7f35b9dbac2d58acR37-R43)

**Review by guidograzioli — APPROVED** [review]
LGTM

**Pull request merged**